index.html

```diff
@@ -0,0 +1,212 @@
+                        </div>
+                    </div>
+                    <div class="item-container">
+                        <div class="item-container2">
+                            <div class="first"><img src="img/program_icon_04 1.png" alt=""></div>
+                            <div class="second"> <p>
+                                Workshop
+                            </p></div>
+                            <div class="third"> <P>Try creating your own work using open source license rather than just watching at it.</P></div>
+                        </div>
+                    </div>
+                    <div class="item-container">
+                        <div class="item-container2">
+                            <div class="first"><img src="img/program_icon_05 1.png" alt=""></div>
+                            <div class="second"> <p>
+                                CC Ignite
+                            </p></div>
+                            <div class="third"> <P>get opportunities to network with CC affiliates around the world, also after the summit.</P></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="button1">
+                    <button class="button1_1">Join CC Global Summit 2015</button>
+                </div>
+                <a href="" class="see-program">SEE THE WHOLE PROGRAM</a>
+            </div>
+
+        </section>
+        <section class="third-page">
+            <div class="third-container">
+                <div>
+                    <div class="second-main">
+                        <p class="speaker">Featured Speakers</p>
+                    <img src="img/title_underline.png" alt="" class="underline">
+
+                    </div>
+                    
+                </div>
+                <div class="speaker-container home-page">
+
+
+                </div>
+
+              
+                <div class="see-more">
+                        <P class="see-more-text">MORE</P>
+                        <img src="img/arrow_down 1.png" alt="">
+
+                </div>
+            </div>
+        </section>
+        <section class="about-page4">
+            <div class="page4-container">
+                <div class="second-main"> 
+                    <p class="second-page-main">Partner</p>
+                    <img src="img/title_underline.png" alt="" class="underline">
+                </div>
+            </div>
+            <div >
+                <div class="page4-container1">
+                    <img src="img/mozilla 1.png" alt="">
+                    <img src="img/googlekorea 1.png" alt="">
+                    <img src="img/naver 1.png" alt="">
+                </div>
+                <div class="page4-container2">
+                    <img src="img/daumkakaologo 1.png" alt="">
+                    <img src="img/airbnb 1.png" alt="">
+    
+                </div>
+
+            </div>
+          
+
+
```

Comment:
- [ ] Your project is coming along nicely! However, I've noticed that there are several instances where there are unnecessary extra spaces between lines of code. It's important to maintain clean and concise code, so I recommend going through your codebase and removing these extra spaces. This practice will enhance readability and ensure that your codebase remains organized. Keep up the good work and strive for a tidy and efficient codebase! 🌟📝🚀